module/zfs/zio.c

```diff
@@ -1424,6 +1420,22 @@ zio_vdev_child_io(zio_t *pio, blkptr_t *bp, vdev_t *vd, uint64_t offset,
 		 */
 		pipeline |= ZIO_STAGE_CHECKSUM_VERIFY;
 		pio->io_pipeline &= ~ZIO_STAGE_CHECKSUM_VERIFY;
+#if defined(__linux__)
```

Comment:
It might still be nice to preserve this as a debugging option for FreeBSD as long as it's disabled by default.  While we'd never expect the verify to fail I don't see the harm in keeping this.

Reply:
Yeah for sure. I went ahead and removed this condition of only being check for Linux. I also reflected this in the man page as well. The default for `zfs_vdev_direct_write_verify_cnt` is now set to `0` for only FreeBSD and `100` for anything else.